lnrpc/rpc.proto

```diff
@@ -1161,6 +1161,13 @@ message Channel {
 
     /// A set of flags showing the current state of the channel.
     string chan_status_flags = 19 [json_name = "chan_status_flags"];
+
+    /// This minimum satoshies this node is required to reseve in its balance    
+    int64 local_chan_reserve = 20 [json_name = "local_chan_reserve"];
```

Comment:
both fields should have a `_sat` suffix to indicate the unit, same for pending channels response

nit: reseve -> reserve, also add period :)

Reply:
Done.